src/cloud/docker/docker-containers-service.md

```diff
@@ -11,21 +11,53 @@ functional_areas:
 The following containers provide the services required to build, deploy and run {{site.data.var.ee}} sites.
 
 {:.bs-callout-info}
-See the [service version values available]({{ site.baseurl }}/cloud/docker/docker-containers.html#service-versions) for use when launching Docker.
+See the [service version values available]({{site.baseurl}}/cloud/docker/docker-containers.html#service-versions) for use when launching Docker.
 
-## Database container
+## Database Container
 
 Container name |Docker base image | Ports exposed
 -------- | -------- | ---------------
 db | [mariadb](https://hub.docker.com/_/mariadb) | 3306 |
 
-When a database container is started for the first time, a new database with the specified name is created and initialized using the configuration variables specified in the docker-compose configuration. The initial start up process also executes all files with extensions .sh, .sql and .sql.gz that are found in the `/docker-entrypoint-initdb.d` directory. Files are executed in alphabetical order. See [mariadb Docker documentation](https://hub.docker.com/_/mariadb)<br>
-<br>To import a database dump into the Database container, place the SQL file into the `.docker/mysql/docker-entrypoint-initdb.d` directory.<br>
+The Database container is based on the [mariadb][db-image] image and includes the following volumes:
 
-Although it is a more complex approach, you can use GZIP to import a database dump by _sharing_ the `.sql.gz` file using the `.docker/mnt` directory and importing the file inside the Docker container.
+-  Volumes:
```

Comment:
In the original list, there was more than just volumes, but don't need this line since it was already introduced in the stem sentence.